lib/commands/search.rb

```diff
@@ -13,7 +13,7 @@ def initialize(repositories:)
     end
 
     def call(args)
-      results = search(args[0])
+      results = search(args.join(" "))
```

Comment:
fix the search to support multiple words in the query. 
also, the search query supports GitHub search syntax now